deployment/ansible/roles/cac-tripplanner.app/tasks/jslibs.yml

```diff
@@ -11,7 +11,7 @@
        state=present
 
 - name: Install NPM dependencies
-  npm: path="{{ root_src_dir }}" state=present
+  command: npm install chdir="{{ root_src_dir }}"
```

Comment:
Yeah, I did remember when I saw the related github discussion between a few of you civic apps folks today. Good to note.